packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh

```diff
@@ -101,11 +111,18 @@ config=$(cat << EOL
   "preimageOracleMinProposalSize": 1800000,
   "preimageOracleChallengePeriod": 86400,
 
-  "fundDevAccounts": false,
+  "fundDevAccounts": $FUNDS_DEV_ACCOUNTS,
   "useFaultProofs": false,
   "proofMaturityDelaySeconds": 604800,
   "disputeGameFinalityDelaySeconds": 302400,
-  "respectedGameType": 0
+  "respectedGameType": 0,
+
+  "usePlasma": $USE_PLASMA,
+  "daCommitmentType": "KeccakCommitment",
+  "daChallengeWindow": 160,
+  "daResolveWindow": 160,
+  "daBondSize": 1000000,
+  "daResolverRefundPercentage": 0
```

Comment:
I'm not sure if there's a use case for it, but you could add `deployCeloContracts` here as well.